### DISCUSSION TITLE
zimaboard/get-started/setup-casaos/

### DISCUSSION THREAD

**Discussion by giscus** [IceWhaleTech/ZimaDocs#2] [ZimaDocs Comments]
# zimaboard/get-started/setup-casaos/

Make sure your computer is connected to the same router as the ZimaBoard

https://docs.zimaboard.com/zimaboard/get-started/setup-casaos/#comments

**Comment by zhangzhongxin** [comment]
从原来的系统升级到0.34 ，访问 http://casaos.local/ 就请求不到了，请问怎么解决

**Comment by zhangzhongxin** [comment]
刚收到ZimaBoard的时候，访问http://casaos.local/ <http://casaos.local/> 可以访问到，按照说明 升级系统 ，再访问 http://casaos.local/ <http://casaos.local/> 就请求不到了。
说明他们确实在一个局域网内，升级 出现了错误 。
我需要知道，怎么连接 初始化 或者其他的快捷，有效的解决方式

**Reply by zhangzhongxin** [reply]
已修复成功。
1:不要从操作界面直接从0.32升级到0.34，可能会出现问题（国内），导致访问不了http://casaos.local，猜测它的执行顺序，没有做网络判断，先将旧版本卸载，再安装新版本，没有先做新版本下载的请求成功还是失败的判断。
2:出现问题可以进入终端，重新安装casaos，需要自己找内网镜像。终端默认 casaos
3:希望尽快完善文档吧
